src/lib/denoify.ts

```diff
@@ -21,7 +22,7 @@ export async function denoify(params: {
 }) {
     process.chdir(params.projectPath ?? ".");
 
-    const srcDirPath = !!params.srcDirPath ? params.srcDirPath : ["src", "lib"].find(sourceDirPath => fs.existsSync(sourceDirPath));
+    const srcDirPath = params.srcDirPath ? params.srcDirPath : ["src", "lib"].find(sourceDirPath => fs.existsSync(sourceDirPath));
```

Comment:
```suggestion
    const srcDirPath = !!params.srcDirPath ? params.srcDirPath : ["src", "lib"].find(sourceDirPath => fs.existsSync(sourceDirPath));
```

Reply:
Is this to reduce unnecessary changes? Or to cast truthy/falsy as boolean to represent real true/false value? I can change this, I am only curious of the reason you want to use double negation (trying to learn how others think)

Reply:
This is debatable indeed, It just makes it more transparent that `srcDirPath` isn't a boolean and  we are looking if it's truethy.  
I try not to rely on implcit bolean convertion. Here I don't know why I didn't wirte `param.srcDirPath !== undefined`...

Reply:
Or we can try `Boolean(params.srcDirPath)`, instead of double negation, otherwise, I would go for `param.srcDirPath !== undefined`